al_servicec.js

```diff
@@ -85,12 +85,18 @@ class AimsC extends m_alUtil.RestServiceClient {
             }
         } catch (e) {
             try {
```

Comment:
could you please double check it is ok to call `try..catch` inside a `catch` block

Reply:
Yea looks to be fine